apps/desks.py

```diff
@@ -321,6 +322,35 @@ def get_desk_name(self, desk_id):
 
         return desk_name
 
+    def get(self, req, lookup):
+        desks_data = super().get(req, lookup)
+
+        desk_list = []
+        members_list = []
+        db_users = app.data.mongo.pymongo('users').db['users']
+
+        # find display_name from the users document for each member in desks document
+        for item in desks_data:
+            desk_list.append(item)
+            for member in item['members']:
+                user_data = db_users.find_one({'_id': member['user']}, {'display_name': 1})
```

Comment:
I think you could get them all at once using `get` with `{'_id': {'$in': [member['user'] for member in item['members']]}}` lookup

Reply:
Do you think it's a good idea? Considering it will return a pymongo cursor object and again I have to iterate over it to get the members_list with unique values from all the desks together.